source/library/Monadoc/Model/Blob.hs

```diff
@@ -49,9 +49,9 @@ migrations =
   ]
 
 new :: ByteString.ByteString -> Blob
-new byteString =
+new contents =
   Blob
-    { contents = byteString,
-      hash = Hash.new byteString,
-      size = ByteString.length byteString
+    { contents = contents,
```

Comment:
This is possible now because `contents` doesn't exist as a selector function. It's kind of nice to be able to do this without having to enable `NamedFieldPuns`. (Although that might be nicer still.)